webkom/lego-webapp
# Individual reactions for meetings

# Description

Adds individual reactions to meeting reactions, meaning you can now see who reacted with which emoji! 🎉

[Backend PR](https://github.com/webkom/lego/pull/3520)

# Result

[Screencast from 08. nov. 2023 kl. 15.37 +0100.webm](https://github.com/webkom/lego-webapp/assets/33326578/feacf562-c627-40f2-b188-a5d8f5abe820)

# Testing


1. Checkout your backend to the [correct branch](https://github.com/webkom/lego/pull/3520)
2. Navigate to a meeting.
3. Verify that adding and removing reactions work.
4. Verify that the reaction tooltip displays who reacted to the emoji.
5. Verify that reactions on articles and quotes work like before, but don't display who reacted.


Resolves [ABA-571](https://linear.app/abakus-webkom/issue/ABA-571/display-who-reacted-on-meetings)
